.github/workflows/nightly_teleport_operational_procedure_snapshot.yml

```diff
@@ -14,14 +14,13 @@ on:
                 required: false
                 default: ''
     pull_request:
-        branches-ignore:
-            - stable/**
```

Comment:
actually, this one doesn't make sense to be executed since it's `stable/8.7` so snapshot of not much use.